src/main/scala/org/broadinstitute/dsde/workbench/gpalloc/service/GPAllocService.scala

```diff
@@ -132,6 +133,15 @@ class GPAllocService(protected val dbRef: DbReference,
     cleanup
   }
 
+  private def overPetLimitOrProjectDeleted(project: String): Future[Boolean] = {
+    googleBillingDAO.overPetLimit(project).recover {
+      case t: GoogleJsonResponseException if t.getStatusCode == 404 => {
+        logger.info(s"Could not locate $project in Google, removing it from database")
```

Comment:
I'm not thrilled with this message being disconnected from the code that will actually try to delete.  Perhaps this wording would be better:

```suggestion
        logger.info(s"Could not locate $project in Google, it should be nuked")
```